src/components/quanitySelector/quanitySelector.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+
+import { Text } from '../text';
+import classes from './quanitySelector.module.scss';
+
+export function QuanitySelector() {
+  const [quanity, setQuanity] = useState(1);
+
+  const decreaseQuanity = () => {
+    if (quanity > 1) setQuanity(quanity - 1);
+  };
+
+  return (
+    <div>
+      <Text
+        variant="poppinsRegular"
+        color="lightGray"
+        className={classes.title}
+      >
+        Quanity
+      </Text>
+      <div className={classes.pickerCont}>
+        <button className={classes.pickerButton} onClick={decreaseQuanity}>
+          &ndash;
+        </button>
+        <Text className={classes.quanity}>{quanity}</Text>
```

Comment:
If I want to buy 20 plants, I have to click 20 times on the top button. It's not good UX, take a look at MUI number input
https://mui.com/base-ui/react-number-input/